### PR TITLE
Merge freifunk Upstream and EOL by grische

### PR DESCRIPTION
**Pull request by T0biii**
merge Freifunk Upstream main (Typescript update)
and EOL by grische (https://github.com/freifunk/meshviewer/pull/81)

### CODE REVIEW

**Review by awlx — APPROVED** [review]
Looks like code

File: lib/config_default.ts [hunk 1]
```diff
@@ -200,21 +366,9 @@ export const config = {
     "AP121U",
     "D-Link DIR-615",
     "D-Link DIR-615 D",
-    "D-Link DIR-825",
```
Comment (grische) [inline lib/config_default.ts]: We will still need all those here

Comment (T0biii) [reply lib/config_default.ts]: i added the D-Link DIR-825 back
i dont know why it was gone.

maybe we should use the upstream branch except the EOL changes and the auto deploy i dont think we got other changes in our repo.

Comment (grische) [reply lib/config_default.ts]: The proper way would be to use the config instead of changing the default values here.

Comment (T0biii) [reply lib/config_default.ts]: that would be done by: https://github.com/freifunkMUC/meshviewer/pull/89

Comment (grische) [reply lib/config_default.ts]: @T0biii sorry, what I meant: We need both the full list of eol and deprecated, similar to what we had before. We have a slightly different firmware (with different/more packages) and hence have slightly different lists of supported devices in comparison to upstream.

**Comment by T0biii** [comment]
i created a new Branch and added there all changes we need for our Meshviewer:
https://github.com/freifunkMUC/meshviewer/tree/new-main

if all looks good we should rename the branch to main and rename the old main to something different

File: lib/config_default.ts (hunk 1 above)
Comment (T0biii) [reply lib/config_default.ts]: i added the devices to the config and the defaults are not touched anymore:
https://github.com/freifunkMUC/meshviewer/blob/new-main/config.json

**Pull request closed without merging**